### DISCUSSION TITLE
Adding the github slack integration to this repo

### DISCUSSION THREAD

**Discussion by astanfel** [Snowflake-Labs/terraform-provider-snowflake#1730] [Ideas]
I would like to be able to follow when a given release happens for the Snowflake Terraform Provider in Slack. Looks like there is a project that can be used for this that I have referenced below: 

reference https://github.com/integrations/slack#subscribe-to-an-organization-or-a-repository

**Comment by sfc-gh-ngaberel** [comment]
Hi @astanfel, this repo is public so I think you should be able to setup these alerts by following the docs, no need to configure anything in this repo. 😃

**Reply by astanfel** [reply]
it doesn't seem to be working for me. I am not sure if I am reference the repo incorrectly. I'll attach some screen shots of what I am trying to do as well as a command that works for the tfe-provider repo as well 

Command I am trying to run: 
<img width="539" alt="Screenshot 2023-04-20 at 10 06 05 AM" src="https://user-images.githubusercontent.com/16292951/233424228-60ba2cca-b610-49fa-a64b-6b6fcfe9c77a.png">

What slack states: 
<img width="680" alt="Screenshot 2023-04-20 at 10 06 12 AM" src="https://user-images.githubusercontent.com/16292951/233424290-99e888e4-1116-49ee-98de-4db74de91f8a.png">

When I try to run a command on a different repo: 
<img width="667" alt="Screenshot 2023-04-20 at 10 05 54 AM" src="https://user-images.githubusercontent.com/16292951/233424450-aaa5a8e2-4f34-45d1-83b8-e0043c5f6cd3.png">

**Reply by astanfel** [reply]
started a discussion the the slack integrations repo as well https://github.com/integrations/slack/discussions/1681

**Reply by sfc-gh-ngaberel** [reply]
Oh interesting, it looks like we might have to add the Slack app to this repo then. I'm not able to do that now but let us know once you've confirmed what needs to happen and we'll see what we can do.

**Reply by ptc-olapovsky** [reply]
Hi @sfc-gh-ngaberel 
As I see you need to add the bot to your Slack workspace by following [https://slack.github.com](https://slack.github.com),
And when you try running the following command `/github subscribe Snowflake-Labs/terraform-provider-snowflake` it will prompt a window that requires to enable the slack bot access to the selected repos in Github,
Then we will be able to subscribe to  Snowflake-Labs/terraform-provider-snowflake as well.

**Reply by ptc-olapovsky** [reply]
Hi @sfc-gh-ngaberel 
Is there an update on enabling slack GitHub bot integration into the repo?

**Reply by astanfel** [reply]
@sfc-gh-ngaberel @sfc-gh-swinkler just wanted to follow up on this to see if y'all had a chance to look into this more

**Comment by harukitake** [comment]
@sfc-gh-asawicki Hi.
Can you check this?
I want to know the releases of this repository soon with slack notification.

**Reply by sfc-gh-asawicki** [reply]
Hey @harukitake. We don't have it currently on our roadmap, I will discuss it with the team, though. I will get back to you as soon as we have any outcomes (from the technical point of view it looks easy but I have other concerns we need to cover before allowing it).

**Reply by harukitake** [reply]
Of course, I think discussion is necessary.
Thank you for considering this!

**Reply by ptc-olapovsky** [reply]
@sfc-gh-asawicki Hi.
Is there any update?

**Reply by sfc-gh-asawicki** [reply]
Hey @ptc-olapovsky, no updates at the moment.

**Comment by sfc-gh-jmichalak** [comment]
Hi all 👋 

The Slack app is installed in the repository. Could you verify if the notifications work on your end?

**Reply by harukitake** [reply]
Thank you for your update!
I have confirmed that I can subscribe this repository.

```
/github subscribe snowflakedb/terraform-provider-snowflake releases
```

**Reply by astanfel** [reply]
Working now! Thanks for getting this added!
